package-lock.json

```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
```

Comment:
This isn't your fault but can you add package-lock.json to your .gitignore file? This is not something that should pushed, I think we have all done it though.

Reply:
should not be pushed**